clang/test/CIR/CodeGen/call.cpp

```diff
@@ -102,3 +104,16 @@ void f12() {
 // LLVM:         %[[#slot:]] = alloca %struct.S, i64 1, align 4
 // LLVM-NEXT:    %[[#ret:]] = call %struct.S @_Z3f10v()
 // LLVM-NEXT:    store %struct.S %[[#ret]], ptr %[[#slot]], align 4
+
+void f13() noexcept;
+void f14() {
+  f13();
+}
+
+// CIR-LABEL: cir.func @_Z3f14v()
+// CIR:         cir.call @_Z3f13v() extra(#[[FN_ATTR]]) : () -> ()
+// CIR:       }
+
+// LLVM-LABEL: define void @_Z3f14v()
+// LLVM:         call void @_Z3f13v()
```

Comment:
Is this capturing the complete LLVM IR. It appears that the incubator doesn't set a callsite attribute here, but classic codegen does. That would seem to indicate that we missed something in lowering.

Reply:
Updated.